soh/soh/SaveManager.cpp

```diff
@@ -324,6 +324,9 @@ void SaveManager::LoadRandomizerVersion2() {
     std::string sariaText;
     SaveManager::Instance->LoadData("sariaText", sariaText);
     randoContext->AddHint(RH_SARIA_HINT, Rando::Hint(RH_SARIA_HINT, {CustomMessage(sariaText)}));
+    std::string loachText;
```

Comment:
This function is old version comparability, it does not need to check for loach as it is new to V3 (unless V2 loach merged without me noticing).

Reply:
It was left from when loach was gonna be merged into v2 before it moved to v3. Removed it.